File: flutter_frontend/print_android_root.go
package main

import (
	"fmt"
	"os"
	"path/filepath"
)

func main() {
	rel := filepath.Join("simple-recipe-browser-30180-30189", "flutter_frontend", "android")
	if st, err := os.Stat(rel); err == nil && st.IsDir() {
		abs, _ := filepath.Abs(rel)
		fmt.Println(abs)
		os.Exit(0)
	}
	fmt.Fprintf(os.Stderr, "ERROR: Android root not found at %s\n", rel)
	os.Exit(1)
}

File: flutter_frontend/print_android_root.py
#!/usr/bin/env python3
import os, sys
root = os.path.join("simple-recipe-browser-30180-30189","flutter_frontend","android")
if os.path.isdir(root):
    print(os.path.abspath(root))
    sys.exit(0)
print("ERROR: Android root not found at {}".format(root), file=sys.stderr)
sys.exit(1)
